pachyderm/values.yaml

```diff
@@ -33,6 +35,8 @@ dash:
     #  cpu: "1"
     #  memory: "2G"
   service:
+    # labels specifies labels to add to the dash service.
+    labels: {}
     # type specifies the Kubernetes type of the dash service.
```

Comment:
Rather than `foo.podLabels` and `foo.service.labels`, I think it would be more consistent to have either `foo.pod.labels` & `foo.service.labels` or `foo.podLabels` or `foo.serviceLabels`. Is there any sort of idiom, best practice or standard in the Helm community at large?

Reply:
Unfortunately Helm has very little in the way of standards and best practices. Based on my experience:
1. `podLabels` appears to be relatively common. It's useful for distinguishing between labels to be added to the pod vs. those to be added to the deployment. I don't recall ever having seen `pod.labels`, however were we to do so here then for the sake of consistency we'd probably want to move `resources` into the same section. IMO it's not worth doing a breaking change for just this feature.
1. `foo.serviceLabels` is much less common than `foo.podLabels`, but that's partially because the ability to add labels to services is much lower priority feature for most charts. `foo.service.labels` is also something I've seen on occasion. The reason I chose to do the latter here was to maintain consistency with how we're doing annotations, eg `foo.service.annotations`. Were we to do `foo.serviceLabels` then we'd probably also want to do `foo.serviceAnnotations`, and that would be a breaking change.

Reply:
podLabels and service.labels seems ok to me. If we end up with more than one value for `pod` then we can make it an object

Reply:
Cool. Just one other note, though: IMHO it is okay to break backwards compatibility as we are currently pre-v1. So feel free to figure out the best way going forward. Again IMHO.

Reply:
Given that there are paying customers using this in production you may wish to consider releasing it as v1. It is much easier to sell a product internally when it it is 1.0 or higher.

Reply:
Thanks @eytanhanig! We'll take that under consideration.